Sources/NIO/PipePair.swift

```diff
@@ -0,0 +1,139 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2019 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+extension NIOFileHandle: Selectable {
+}
```

Comment:
I assume we don't need this?

Reply:
We do. For `PipePair` to be a `SocketProtocol` implementation, it's `SocketType` must conform to `Selectable`. `PipePair'`s `SocketType` however is `NIOFileHandle`.